_posts/2020-11-30-developer-survey-2020.md

```diff
@@ -0,0 +1,99 @@
+
+![Backend languages](/img/developer-survey-2020/backend_languages.png)
+
+Moving on to more focused questions, Java proved to be the most used language for backend development. 45% reported Java being one of the primary backend languages in ther projects. Rounding out top 3, are then Clojure with about 19% usage and Kotlin with 18%. Compared to results from 2018, percentage of Kotlin users rose from 15% to 24% (comparison based on the question of all languages used). Java and Clojure usage were both a little bit down when comparing 2020 to 2018, which could be explained by the increased usage of Clojure. Neither seem to be going away though, as shown by being top 1 and 2 used languages for backend and both are well liked (more on that later).
+
+As this question was about backend languages, we can assume that JavaScript, TypeScript and ClojureScript here mean that they are run on NodeJS. TypeScript was then the most used language with only a slight margin over JavaScript (11% vs. 10%)
+
+![Frontend languages](/img/developer-survey-2020/frontend_languages.png)
+
+For frontend languages the clear number one was JavaScript with 68% usage. The other languages that have wide usage are Typescript (37%) and ClojureScript (19%). ClojureScript is an obvious choice for projects that use Clojure in the backend, so it seems those two go hand in hand quite much. CoffeeScript has a handful of users - it seems that the number has been stable compared to 2018 figures (4% overall usage in 2020 compared to 5% in 2018). Most likely it's not used for any new projects but it is still in use in some long-time project(s). PureScript also had a handful users here, which seems to be new compared to 2018.
+
+## Languages we like
+![Liked languages](/img/developer-survey-2020/liked_languages.png)
+
+Next, we asked for opinions: what language do you like the most? The most used backend language was also the most liked one, as 17% of respondents said that Java was their favorite language. Clojure came second with 15%, which is impressive considering that even though it is used quite much, in terms of raw numbers Java developers outnumber Clojure developers by a mile. Clojure developers then really know what they want. Kotlin was the third most liked language, meaning that the top 3 of liked languages mirrors the top 3 of backend languages by usage.
+
+Compared to results from 2018, JavaScript fell down from number #2 spot to number #4. This could be explained TypeScript, which gained almost as many percents now than JavaScript lost. All in all, most top languages were in decline compared to the previous survey, except Kotlin, which received much more love this year compared to 2018. This time there also seemed to be a lot more niche languages getting a few votes than before, which also explains why the top languages were a bit in decline.
+
+## Languages we would like to use
+![Languages we would like to use](/img/developer-survey-2020/would_like_to_use_languages.png)
```

Comment:
This image is visually a bit boring. Could it be visually more in line with the pictures above?

Reply:
This is still a placeholder